### PR TITLE
Update NOTM stage to match dev with same env vars and structure.

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: notm/stage/main.tf [hunk 1]
```diff
@@ -212,3 +216,46 @@ resource "azurerm_dns_cname_record" "cname" {
   record              = "${var.app-name}.azurewebsites.net"
   tags                = "${var.tags}"
 }
+
+resource "azurerm_template_deployment" "elite2-webapp" {
```
Comment (glenjamin) [inline notm/stage/main.tf]: I don't think we need these bits - this was from when I did that elite2-in-azure proof-of-concept that didn't work very well.

Comment (Mjwillis) [reply notm/stage/main.tf]: so everything below can go?

Comment (glenjamin) [reply notm/stage/main.tf]: Yep, should probably also delete the one in dev actually.

**Comment by glenjamin** [comment]
Has the elite2 app service also been deleted? Annoyingly because it was deployed by template deleting via terraform doesn't delete the underlying resource.

**Pull request merged**